### ISSUE TITLE
host-local IPAM plugin doesn't work with kubernetes

### ISSUE THREAD

**Issue by tomdee** [appc/cni#126]
When using the CNI plugin with Kubernetes, a number of additional arguments are [provided](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni.go#L201). This causes the `host-local` plugin to exit with

```
2016-02-17 19:03:39,885 28126 [default/busybox] INFO Using IPAM plugin at: /opt/cni/bin/host-local
2016-02-17 19:03:39,888 28126 [default/busybox] ERROR IPAM plugin error (rc=1): {
    "code": 100,
    "msg": "ARGS: invalid key \"K8S_POD_NAMESPACE\""
}
```

**Comment by steveeJ** [comment]
A discussion about this has started in #109, and I agree that the situation is not ideal. However I would be very unhappy to just ignore unknown arguments all together. IMO, we need a better strategy than what we have and what's currently proposed.

**Comment by rkamudhan** [comment]
@steveeJ @tomdee This is issue is still prevail in Kubernetes > 1.4.0. I check it with Kubernetes version 1.3.6, it is working fine. Whether you guys experience same issue with Kubernetes > 1.4.0 ?

**Comment by tomdee** [comment]
I thought this was fixed by kubernetes/kubernetes#24983
